static/js/place/place_template.tsx

```diff
@@ -556,8 +557,18 @@ class ChartBlock extends Component<ChartBlockPropType, unknown> {
     );
   }
 
-  // TODO(shifucun): Add more config to indicate whether to use perCapita for
-  // place comparison.
+  private copyAndUpdateConfig(config: ConfigType) {
+    const conf = { ...config };
+    conf.chartType = chartTypeEnum.GROUP_BAR;
+    conf.axis = "PLACE";
+    if (conf.canScale == null || conf.canScale) {
```

Comment:
this should update if there conf.denominator is defined (or we can revert the extra canScale from the config - i prefer this option)